nestjs-prisma/README.md

```diff
@@ -0,0 +1,102 @@
+# Nx + NestJs + Prisma
```

Comment:
Can you also add the badges as described in the CONTRIBUTING guide
https://github.com/nrwl/nx-recipes/blob/main/CONTRIBUTING.md